packages/@react-spectrum/checkbox/package.json

```diff
@@ -41,7 +41,9 @@
     "@react-aria/interactions": "^3.21.1",
     "@react-spectrum/form": "^3.7.4",
     "@react-spectrum/label": "^3.16.4",
+    "@react-spectrum/layout": "^3.6.3",
     "@react-spectrum/utils": "^3.11.5",
+    "@react-spectrum/view": "^3.6.8",
```

Comment:
Where did these new deps come from? Seems to come up in a couple of the package.json changes, were they auto added by the IDE perhaps? The only imports for these are in the stories and stuff so we don't need them here right?